student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(_name, age, _parent_permission, classroom)
+    super('Unknown', age, true)
```

Comment:
- [ ] Please kindly declare the parameters inherited from the `Person.rb` appropriaely as you did in `Person.rb` file. The `super()` function is used to call methods and constructors of the parent class, kindly ensure that the parameters are written correctly. 👍